hack/secret-creator/create-plnsvc-secrets.sh

```diff
@@ -64,6 +65,48 @@ stringData:
 EOF
 }
 
+create_db_cert_secret_and_configmap() {
+    echo "Creating Postgres TLS certs" >&2
+    if kubectl get secret -n tekton_results postgresql-tls &>/dev/null; then
+        echo "Postgres DB cert secret already exists, skipping creation"
+        return
+    fi
+    mkdir -p .tmp/tekton-results
+    openssl req -new -nodes -text \
+        -keyout ".tmp/tekton-results/ca.key" \
+        -out ".tmp/tekton-results/ca.csr" \
+        -subj "/CN=cluster.local" \
+        > /dev/null
+    chmod og-rwx ".tmp/tekton-results/ca.key"
+    openssl x509 -req -days 7 -text -extensions v3_ca \
+        -signkey ".tmp/tekton-results/ca.key" \
+        -in ".tmp/tekton-results/ca.csr" \
+        -extfile "/etc/ssl/openssl.cnf" \
+        -out ".tmp/tekton-results/ca.crt" \
+        > /dev/null
+    openssl req -new -nodes -text \
+        -subj "/CN=postgres-postgresql.tekton-results.svc.cluster.local" \
+        -addext "subjectAltName=DNS:postgres-postgresql.tekton-results.svc.cluster.local" \
+        -out ".tmp/tekton-results/tls.csr" \
+        -keyout ".tmp/tekton-results/tls.key" \
+        > /dev/null
+    chmod og-rwx ".tmp/tekton-results/tls.key"
+    openssl x509 -req -text -days 7 -CAcreateserial \
+        -extfile <(printf "subjectAltName=DNS:postgres-postgresql.tekton-results.svc.cluster.local") \
+        -in ".tmp/tekton-results/tls.csr" \
+        -CA ".tmp/tekton-results/ca.crt" \
+        -CAkey ".tmp/tekton-results/ca.key" \
+        -out ".tmp/tekton-results/tls.crt" \
+        > /dev/null
+    cat ".tmp/tekton-results/ca.crt" > ".tmp/tekton-results/tekton-results-db-ca.pem"
```

Comment:
Do we only need ca.crt? My understanding is to combine both ca.crt and tls.crt

Reply:
The client only needs the ca.crt (which signed the server.crt), so it can verify the server.crt. From the documentation:
```server.crt and server.key should be stored on the server, and root.crt should be stored on the client so the client can verify that the server's leaf certificate was signed by its trusted root certificate. root.key should be stored offline for use in creating future certificates.```

Reply:
Understood